wiki/Welcome/en.md

```diff
@@ -1,11 +1,11 @@
 # Welcome
 
-Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; osu!mania, a key smashing mania.
+Welcome to osu!, a free-to-win rhythm game developed by peppy with four game modes: osu!standard, a circle clicking simulator; osu!taiko, a drumming emulator; osu!catch, a fruit salad catcher; and osu!mania, a key smashing synthesizer.
```

Comment:
Link `peppy` to https://osu.ppy.sh/users/2.

Reply:
I don't think the lead paragraph for the welcome page should have any links.